pkg/constants/constants.go

```diff
@@ -175,7 +175,6 @@ const (
 
 	FromIndex   = "registry-proxy.engineering.redhat.com/rh-osbs/iib-preview-rhtap:{{ OCP_VERSION }}"
```

Comment:
Can we move `FromIndex` and `TargetIndex` to release/const.go file to make it comply with the test case selection mechanism？ Because there is no other test case use them except the release test cases.

Reply:
Could this be done in a separate ticket? It seems to me there are more stuff regarding FBC that should be moved, not only the "indexes", and also some stuff will be removed by another PR as well.

Reply:
@jinqi7 can you lgtm this one as well? The code is already in prod.

Reply:
Can you create a ticket about  "more stuff regarding FBC" need to be moved and "some stuff" need to be moved?  In case, it affect the test coverage for the PRs in the future.

Reply:
it is being removed by this other PR https://github.com/konflux-ci/e2e-tests/pull/1412/files.

Reply:
sorry, misread the "moved" as removed. I need to crosscheck. It was long since I created this PR, so I just don´t remember what it is.  Once I see that I can add the ticket.